x-pack/plugin/sql/qa/src/main/resources/command.csv-spec

```diff
@@ -60,7 +61,8 @@ DAY_OF_WEEK      |SCALAR
 DAY_OF_YEAR      |SCALAR
 DOM              |SCALAR
 DOW              |SCALAR         
-DOY              |SCALAR         
+DOY              |SCALAR
+FORMAT           |SCALAR
```

Comment:
`FORMAT` seems a bit too generic - is there any DB that uses it?

Reply:
SQL Server